tests/unit/algo/detection/heads/test_custom_anchor_generator.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""Test of CustomAnchorGenerator."""
+
+import pytest
+import torch
+from otx.algo.detection.heads.custom_anchor_generator import SSDAnchorGeneratorClustered
+
+
+class TestSSDAnchorGeneratorClustered:
+    @pytest.fixture(autouse=True)
+    def setup(self) -> None:
```

Comment:
Likewise this https://github.com/openvinotoolkit/training_extensions/pull/2708#discussion_r1420025955, there are so many anti-patterns in this PR for using pytest. There is no reason to use `pytest.fixture` in this way. You can just declare it as the member variable if you use it like this.

Please fix everywhere in this PR if you use pytest fixture in this way.

Reply:
Done by [06bf724](https://github.com/openvinotoolkit/training_extensions/pull/2711/commits/06bf7249cc4c8d47b5bec215c1fb3ef8d9929022)